identity/src/main/java/com/android/identity/DeviceResponseGenerator.java

```diff
@@ -100,9 +100,9 @@ public DeviceResponseGenerator(@Constants.DeviceResponseStatus long statusCode)
         MapBuilder<CborBuilder> insOuter = issuerNameSpacesBuilder.addMap();
         for (String ns : issuerSignedData.keySet()) {
             ArrayBuilder<MapBuilder<CborBuilder>> insInner = insOuter.putArray(ns);
-            for (byte[] encodedIssuerSignedItemBytes : issuerSignedData.get(ns)) {
-                DataItem issuerSignedItemBytes = Util.cborDecode(encodedIssuerSignedItemBytes);
-                insInner.add(issuerSignedItemBytes);
+            for (byte[] encodedIssuerSignedItem : issuerSignedData.get(ns)) {
+                // We'll do the #6.24 wrapping here.
```

Comment:
Not sure this comment adds anything helpful. Where else would it be?

Reply:
I'll leave it in for now, want to land this thing.